t/134-worker-FFI.t

```diff
@@ -0,0 +1,116 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+use lib 'lib';
+use Test::Nginx::Socket::Lua;
+
+#worker_connections(1014);
+#master_on();
+workers(3);
+#log_level('warn');
+
+repeat_each(2);
+#repeat_each(1);
+
+plan tests => repeat_each() * (blocks() * 3);
+
+#no_diff();
+#no_long_string();
+run_tests();
+
+__DATA__
+
+=== TEST 1: content_by_lua + ngx_http_lua_ffi_worker_pid()
```

Comment:
I suggest we should avoid testing these FFI C API functions specifically here. The tradition is to run the test suite of ngx_lua with lua-resty-core enabled since lua-resty-core reimplements the same Lua API with these FFI C functions under the hood already.